web-client/src/presenter/state-public.ts

```diff
@@ -49,6 +49,7 @@ export const baseState = {
   },
   isPublic: true,
   isTerminalUser: false,
+  modal: {},
```

Comment:
Does the modal work on mobile?

Reply:
Indeed! Tenille was able to see & use it on a mobile device on test: https://flexion.slack.com/archives/CD6DEU37Z/p1718387128439669

Reply:
Sweetness